cluster/cluster.pb.go

```diff
@@ -61,7 +66,7 @@ func (x *CID) ProtoReflect() protoreflect.Message {
 
 // Deprecated: Use CID.ProtoReflect.Descriptor instead.
 func (*CID) Descriptor() ([]byte, []int) {
-	return file_cluster_proto_rawDescGZIP(), []int{0}
+	return file_cluster_cluster_proto_rawDescGZIP(), []int{0}
```

Comment:
source code change in generated code.

Reply:
Changed. I generate the files as they have been initially.